packages/block-library/src/navigation/theme.scss

```diff
@@ -3,4 +3,45 @@
 	ul li {
 		list-style: none;
 	}
+
+	ul > li > ul {
+		background: #e3e3e3;
+		border-radius: 4px;
```

Comment:
I wish we had some sort of system for cataloguing these opinionated styles. Like, a consistent coherent visual identity for all WordPress opinionated styles that blocks can opt into. Right now they are kind of designed in isolation, but it'd be nice to have a system similar to the variables sheet we have for UI.

Things like color variables, border radii, paddings, etc. Not for this PR, but CC: @ItsJonQ as I think you've been vaguely exploring things like these?

Reply:
@jasmussen Ah yes! I outlined my idea here:
https://github.com/WordPress/gutenberg/issues/9534#issuecomment-555173108